src/createItems.js

```diff
@@ -26,20 +20,27 @@ function createItems(tasks) {
       listItem.classList.toggle('completed', this.checked);
     });
 
-    listItem.appendChild(checkbox);
-
     const description = document.createElement('span');
     description.textContent = task.description;
+    description.classList.add('description');
+
+    const ellipsis = document.createElement('span');
+    ellipsis.textContent = '...';
+    ellipsis.classList.add('ellipsis');
+
+    const itemContainer = document.createElement('div');
+    itemContainer.classList.add('item-container');
+    itemContainer.appendChild(checkbox);
+    itemContainer.appendChild(description);
 
-    listItem.appendChild(description);
+    listItem.appendChild(itemContainer);
+    listItem.appendChild(ellipsis);
 
-    if (task.completed) {
-      listItem.classList.add('completed');
-    }
     todoList.appendChild(listItem);
 
     const hr = document.createElement('hr');
     todoList.appendChild(hr);
   });
 }
-export { createItems, tasks };
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] As i have suggested above, you are not meant to use any linters diable rule in your codebase. The linters are there to make sure your code meets the best standard practices. I'll suggest you remove this linter diable rule and fix the issue. :+1: :+1: :+1: